src/applications/find-forms/containers/SearchForm.jsx

```diff
@@ -10,91 +10,81 @@ import {
 } from '../helpers/selectors';
 import { fetchFormsThunk } from '../actions';
 
-export class SearchForm extends Component {
-  static propTypes = {
-    // From mapStateToProps.
-    fetching: PropTypes.bool.isRequired,
-    useLighthouseSearchAlgo: PropTypes.bool,
-    // From mapDispatchToProps.
-    fetchForms: PropTypes.func.isRequired,
-  };
-
-  constructor(props) {
-    super(props);
-
-    // Derive the current query params.
-    const queryParams = new URLSearchParams(window.location.search);
-
-    // Derive the query.
-    const query = queryParams.get('q') || '';
+export const SearchForm = ({
+  fetchForms,
+  fetching,
+  useLighthouseSearchAlgo,
+}) => {
+  // Derive the current query params.
+  const queryParams = new URLSearchParams(window.location.search);
+  // Derive the query.
+  const query = queryParams.get('q') || '';
+  const [queryState, setQueryState] = useState({
```

Comment:
Any reason to keep queryState as an object and not just have query as a string? 🙂

Reply:
Oh that's a thing. I forgot you can do that. 
Old habits with state being an object.